### DISCUSSION TITLE
opening folder and creating folder in brave browser is not available

### DISCUSSION THREAD

**Discussion by ShantanuXD** [phcode-dev/phoenix#880] [Q&A] [answered]
opening folder and creating folder in brave browser is not available
can u tell me a fix for that
![image](https://user-images.githubusercontent.com/114171586/212275249-713de9eb-d083-4241-8c4b-2650d1a58c7c.png)

**Comment by danjguzman** [comment]
I was about to post this. I also use Brave, and I get the same issue.

**Comment by abose** [comment]
Sorry for the late reply. Just found this thread. Brave unfortunately do not support the feature. Please make some noise here to enable the feature https://github.com/brave/brave-browser/issues/11407

We are working on a native build to be available in 3 months. Or you could use chrome or edge or opera for now to edit local folders

**Comment by danjguzman** [comment, marked as answer]
Brave does support the feature, it just needs to be manually _enabled_ with a flag.

1. In the URL bar enter **brave://flags/**
2. Search for **File System Access API**.
3. Select **Enabled**.
4. **Restart** Brave.

I would much prefer a _native_ application for Mac OS or Windows instead of a browser since browsers are prone to crashing if a tab goes awry. 

![Screen Shot 2023-02-14 at 1 15 33 PM](https://user-images.githubusercontent.com/33531668/218864919-ea50b7e4-3f3e-40eb-995e-9ce9ad3b8560.png)

**Reply by abose** [reply]
Thanks for the tip. Marked as answer.
